### DISCUSSION TITLE
Please consider adding a measure to clean up the staging files left behind when upgrading the game.

### DISCUSSION THREAD

**Discussion by Iovenoname** [Klocman/Bulk-Crap-Uninstaller#607] [Ideas]
I have a very good idea, I hope bulk-crap-uninstaller will add in the next iteration to clean up the files that are left behind by the update of networked games, they take up a lot of space on the PC ( e.g. League of Legends , Dungeon and Fighter, World of Warcraft ) these games leave a lot of cache files after each update, cleaning them up is very difficult for the average user so I propose to add in a cache file cleaner for the games that are left behind. Warcraft ) These games leave a huge amount of cache files on the PC after each update, and cleaning them up is very difficult for the average user, so I propose to add a cleanup of the game's staging files.

**Comment by Klocman** [comment]
Hello, and thank you for the suggestion!

This would require to add and maintain code specific for each of those games, which is not really viable in a general uninstaller like BCU. Most people won't be able to even test this code since they do not own or play those games.

It would be a lot more viable if there were third party open-source tools that already provide this feature, as they could be integrated, but I don't know of any and as such this is not likely to be implemented.

**Reply by Iovenoname** [reply]
Thank you for your reply! I thought about it after reading your message, and I think your concerns are correct. Doing so does require maintaining specific code and consuming time, which brings extra burden. I am sorry. I like open source very much. I think it's great that they allow anyone to review it and that it helps build trust and spread the internet to poorer parts of the world. Your project is great. I love it. Have a great day. God bless you
